spec.md

```diff
@@ -83,9 +83,9 @@ For UCAN.storage `att` MUST contain at least one capability using the `storage`
 ]
 ```
 
-# 4. `storage` scheme
+# 4. `storage` Capability
 
-The `storage` scheme represents the ownership and access permissions over storage resources in a [Service](#21-Service).
+The `storage` capability represents the ownership and access to operations over storage resources in a [Service](#21-Service).
 
 ## 4.1 Resource Pointer
 
```

Comment:
A quick question around this [line](https://github.com/nftstorage/ucan.storage/pull/3/files#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545L96).

`MUST add another path segment to the resource URI` is this a requirement?
Shouldn't we allow to delegate the same resource. ie. I'm creating a UCAN to delegate all the capabilities on the whole "bucket" to a did (ie for the cli)

Reply:
yes that's correct need to update the spec

Reply:
i think this has been handled but not sure. if not we should make an issue.

Reply:
Indeed!